### PR TITLE
Desktop

### PR DESCRIPTION
**Pull request by dagmawiii03**
**For the fourth project in your portfolio website:** 

- Use media queries to make sure that your website displays correctly on desktop screens
- Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config)
- Made sure that you used correct github flow
- made sure that your dekstop version looks like the one in the Figma design template that you chose
- Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -2,678 +2,526 @@
 /* stylelint-disable no-missing-end-of-source-newline */
```
Comment (huseyinbiyik) [inline css/styles.css]: Please don't disable any linters. They are important for an error-free standardized codebase. These linter can be fixed automatically if you set up linter on your local and `npx stylelint --fix` command on your terminal. Check the setup instructions from this [link](https://github.com/microverseinc/linters-config/tree/master/html-css).

File: index.html [hunk 2]
```diff
@@ -10,346 +10,305 @@
...
-      <!--landing pages-->
-      <section class="pages">
-        <section class="header">
-          <h3>
-            My Recent <br />
-            Works
-          </h3>
+    </section>
+    <section class="recent-works">
+      <h1>My Recent Works</h1>
```
Comment (huseyinbiyik) [inline index.html]: Kindly add space after My Recent Work like on your main branch. Optionally, you can add green line after that as well. 

Screenshot 2023-02-26 at 12 07 42 AM
Screenshot 2023-02-26 at 12 07 51 AM

File: index.html [hunk 3]
```diff
@@ -10,346 +10,305 @@
...
+      </section>
+    </header>
+    <!-- home page -->
+    <section class="home-page">
       <nav>
-        <ul class="menu">
+        <ul class="desktop-navigation">
           <li>Portfolio</li>
           <li>About</li>
           <li>Contact</li>
```
Comment (huseyinbiyik) [inline index.html]: Kindly use an element for the Portfolio, About and Contact elements here. They need to go the related area on the page. You can use anchor tag to do that.
- Also please make sure font-size and font-weight matches with the Figma template.
Screenshot 2023-02-26 at 12 02 16 AM
Screenshot 2023-02-26 at 12 02 57 AM

File: css/styles.css [hunk 4]
```diff
@@ -2,678 +2,526 @@
...
   border-radius: 8px;
-  background: #ebf0ee;
 }
 
-.grid1 {
-  background-image: url(../images/background-contact-shape2.png);
-  height: 300px;
-  border-radius: 5px;
+.card-work > .card-tags > li {
+  background: rgb(226, 225, 225);
```
Comment (huseyinbiyik) [inline css/styles.css]: Kindly fix the color of the background for the cards-tags here like on the Figma template.
Screenshot 2023-02-26 at 12 04 38 AM

File: index.html [hunk 5]
```diff
@@ -10,346 +10,305 @@
...
+              ></textarea>
+            </li>
+            <li class="submit-button">
+              <button type="submit">Get in touch</button><br />
+              <small></small>
+            </li>
+          </ul>
+        </form>
+      </section>
+      <ul class="social-media">
```
Comment (huseyinbiyik) [inline index.html:281]: Kindly add the gray line before the social media icon.
Screenshot 2023-02-25 at 11 57 55 PM

File: index.html [hunk 6]
```diff
@@ -10,346 +10,305 @@
...
+        <li>
+          <a href="">
+            <img
+              src="./assets/social-media/angellist.svg"
+              alt="angellist icon"
+            />
+          </a>
+        </li>
+      </ul>
+      <a href="www.gmail.com" class="link-mail">dagmawi.yoha03@gmail.com</a>
```
Comment (huseyinbiyik) [inline index.html:314]: Please hide the mail adress on the desktop view like on the Figma template.
Screenshot 2023-02-26 at 12 00 02 AM
Screenshot 2023-02-26 at 12 00 12 AM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @dagmawiii03,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Perfectly responsive elements 👍 👍 
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- You have great commit messages, but please remember to capitalize the first word of the commit messages 👍 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 1 above)
Comment (dagmawiii03) [reply css/styles.css]: Done

File: index.html (hunk 3 above)
Comment (dagmawiii03) [reply index.html]: Done

File: css/styles.css (hunk 4 above)
Comment (dagmawiii03) [reply css/styles.css]: Done

File: index.html (hunk 6 above)
Comment (dagmawiii03) [reply index.html:314]: Done

File: index.html (hunk 2 above)
Comment (dagmawiii03) [reply index.html]: Done

File: index.html [hunk 7]
```diff
@@ -10,346 +10,308 @@
...
       </section>
-      <!--landing pages-->
-      <section class="pages">
-        <section class="header">
-          <h3>
-            My Recent <br />
-            Works
-          </h3>
+    </section>
+    <section class="recent-works">
```
Comment (franclobo) [inline index.html:100]: - **[OPTIONAL]** Please 🙏🏽 add `id="recent-works"` to allow the link in the navbar to go to the corresponding section for a better UX. Also for the about and contact sections.

File: index.html [hunk 8]
```diff
@@ -10,346 +10,308 @@
...
+    <section class="home-page">
       <nav>
-        <ul class="menu">
-          <li>Portfolio</li>
-          <li>About</li>
-          <li>Contact</li>
+        <ul class="desktop-navigation">
+          <li><a href="#recent-works">Portfolio</a></li>
+          <li><a href="#about-me">About</li>
+          <li><a href="#about-me">Contact</li>
```
Comment (franclobo) [inline index.html:49]: - **[OPTIONAL]** Well done adding the `href` to the links 👏🏽 but there is a mistake on the Contact link the `href `should go to the contact form section.

**Review by franclobo — APPROVED** [review]
Hi @dagmawiii03, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No inter errors ✔️ 
- Better practices of HTM & CSS ✔️ 
- Correct use of media queries 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abel-blue — APPROVED** [review]
# Approved
approved

**Pull request merged**